packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -831,6 +838,56 @@ - (void)invalidateLayer
   }
 
   [_boxShadowLayer removeFromSuperlayer];
```

Comment:
this might be result of a merge that I committed recently, but can you put your code above this line so box shadow logic is together

Reply:
my bad i missed that, fixed.